README.md

```diff
@@ -48,7 +48,9 @@ First, you will need to customize and improve your document retrieval project on
 
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - [nvm](https://github.com/nvm-sh/nvm#installation-and-update)
+  - Set Node version to 16.x.x (e.g. `16.14.2`) via `nvm install 16.14.2` and `nvm use 16.14.2`
 - [yarn](https://yarnpkg.com/getting-started/install) or [npm](https://www.npmjs.com/get-npm)
+- For MacOS, install any waiting OS Software Updates and run `xcode-select --install` in terminal
```

Comment:
```suggestion
- On MacOS, the Xcode Command Line Tools. To install, run `xcode-select --install` in terminal.
```

Why do we need to specify running updates? I think just `Xcode-select --install` should be fine (even if user is down level on the OS).

Reply:
When trying to get it working recently, `xcode-select --install` points to running `Software update` first if there are updates available.